packages/core-suggest/core-suggest.js

```diff
@@ -50,7 +50,7 @@ export default class CoreSuggest extends HTMLElement {
   get ajax () { return this.getAttribute('ajax') || '' } // Always return string consistent with .value or .className
   set ajax (url) { this.setAttribute('ajax', url) }
 
-  get limit () { return Number(this.getAttribute('limit')) || Infinity }
+  get limit () { return Math.max(0, Number(this.getAttribute('limit'))) || Infinity }
```

Comment:
Why max here? :)

Reply:
to coerce negative numbers to 0

Reply:
Ah, yeah, smart :) then `Number` is not needed :)
`Math.max(0, this.getAttribute('limit'))`